static/app/components/searchQueryBuilder/tokens/freeText.tsx

```diff
@@ -541,7 +542,7 @@ function SearchQueryBuilderInputInternal({
             resetInputValue();
             trackAnalytics('search.key_manually_typed', {
               organization,
-              search_type: savedSearchType === 0 ? 'issues' : 'events',
+              search_type: recentSearches === 0 ? 'issues' : 'events',
```

Comment:
is there a reason why this isn't `recentSearchTypeToLabel(recentSearches)`

Reply:
Missed this one - thanks!